modules/dag-l1/src/main/scala/org/tessellation/dag/l1/domain/transaction/package.scala

```diff
@@ -0,0 +1,5 @@
+package org.tessellation.dag.l1.domain
+
+package object transaction {
+  val transactionLoggerName = "org.tessellation.dag.l1.domain.TransactionLogger"
```

Comment:
Should we name the logger as a class that doesn't exist? Maybe we could call it `TransactionLogger` simply? Or you could actually create an Object `TransactionLogger` at the given package and have a function for instantiating that logger to be used wherever you need it. You will not have to import the name and get the logger by name everywhere and the class will actually exist.

Reply:
I made it the same way as the RumorLogger. I think it should be any arbitrary name, it can be just `TransactionLogger` as well I think. @tbekas wdyt?

Reply:
I agree, let's use an arbitrary name, rather than non-existing class name. @marcinwadon could you change the logger in gossip to `RumorLogger` as well?

Reply:
done :+1: